leases/tests/test_lease_models.py

```diff
@@ -349,6 +350,46 @@ def test_berth_lease_is_not_active_after_today():
     assert not BerthLease.objects.get(id=lease.id).is_active
 
 
+@freeze_time("2020-06-01T08:00:00Z")
+def test_winter_storage_lease_is_active_before_season():
+    lease = WinterStorageLeaseFactory(status=LeaseStatus.PAID)
+
+    assert WinterStorageLease.objects.get(id=lease.id).is_active
+
+
+@freeze_time("2020-10-01T08:00:00Z")
+def test_winter_storage_lease_is_active_during_season():
+    lease = WinterStorageLeaseFactory(status=LeaseStatus.PAID)
+
+    assert WinterStorageLease.objects.get(id=lease.id).is_active
+
+
+@freeze_time("2020-10-01T08:00:00Z")
+def test_winter_storage_lease_is_not_active_status():
+    lease = WinterStorageLeaseFactory(status=LeaseStatus.EXPIRED)
+
+    assert not WinterStorageLease.objects.get(id=lease.id).is_active
+
+
+@freeze_time("2020-10-01T08:00:00Z")
+def test_winter_storage_lease_is_not_active_before_today():
+    today = date.today()
+    start_date = today - relativedelta(days=5)
+    end_date = today - relativedelta(days=1)
+    lease = WinterStorageLeaseFactory(
+        status=LeaseStatus.PAID, start_date=start_date, end_date=end_date
+    )
+
+    assert not WinterStorageLease.objects.get(id=lease.id).is_active
+
+
+@freeze_time("2020-07-10T08:00:00Z")
+def test_winter_storage_lease_is_not_active_after_today():
+    start_date = date.today()
+    lease = WinterStorageLeaseFactory(status=LeaseStatus.PAID, start_date=start_date)
+    assert not WinterStorageLease.objects.get(id=lease.id).is_active
```

Comment:
hm, interesting catch, I think this case should be `true` 😅 since if the lease starts today, then yeah, it is active today and untill the day it ends.

Reply:
🤔 fair point